_docs-v3/date-navigation/next.md

```diff
@@ -19,7 +19,7 @@ Example using `next` with an external button:
 
 ```js
 $('#my-next-button').click(function() {
-  $('#calendar').fullCalendar('next');
+  new Calendar.next();
```

Comment:
this is not syntactically correct. it would be `(new Calendar()).next();` instead.

however, for examples that demonstrate a method call, let's just do `calendar.next()`. we'll assume there's a globally available `calendar` object.